### ISSUE TITLE
Updating Workflow-modules

### ISSUE THREAD

**Issue by tonygardella** [PecanProject/pecan.book#16]
Currently the workflow.modules page is consistent with PEcAn 1.2.6. This needs to updated. 

I will take a first pass on this, then pass it off to others to get better explanations of certain modules.

**Comment by mdietze** [comment]
An important question is whether we actually want to update that or just rely on updating the automated package documentation, which is also on the web (see links off one of the wiki main pages).  I tend to think we should just document the very high-level structure and then leave the function level documentation to Roxygen, otherwise the documentation will always be out of date and duplicated

**Comment by tonygardella** [comment]
I agree with keeping it higher level. My goal for the page would be to educate the user about the steps that show up under the STATUS  of a run and help demystify workflow.Rout so that if a run halts they know which package documentation they should consult for help.
